File: python/1.py
# project euler #1 in python by antekone
# http://anadoxin.org/blog
#
# usage:
#
# $ python3 1.py

sum = 0
for i in range(0, 1000):
    if i % 5 == 0 or i % 3 == 0:
        sum += i

print(sum)

File: lua/1.lua
-- project euler #1 in lua by antekone
-- http://anadoxin.org/blog
--
-- usage:
--
-- $ lua 1.lua

sum = 0
for i = 0, 999 do
	if (i % 3 == 0) or (i % 5 == 0) then
		sum = sum + i
	end
end

print(sum)
